### PR TITLE
android executor tests

### PR DESCRIPTION
**Pull request by alexander-kulikovskii**

### CODE REVIEW

File: lib/src/main/java/fi/epicbot/toster/model/SwipeMove.kt [hunk 1]
```diff
@@ -87,4 +87,4 @@ internal fun SwipeMove.toMove(
 }
 
 private fun provideOffset(swipeOffset: SwipeOffset, totalValue: Int): Int =
-    max((totalValue * swipeOffset.offsetFactor).toInt(), swipeOffset.offsetPx)
+    min((totalValue * swipeOffset.offsetFactor).toInt(), swipeOffset.offsetPx)
```
Comment (HlebMaliborski) [inline lib/src/main/java/fi/epicbot/toster/model/SwipeMove.kt:90]: Next PR will be max again?)

File: lib/src/main/java/fi/epicbot/toster/model/Action.kt [hunk 2]
```diff
@@ -55,7 +55,7 @@ internal fun Action.title(): String {
         Action.ClearAppData -> "Clear app data"
         Action.ClearText -> TODO()
         is Action.Click -> "Click to ($x;$y)"
-        Action.ClickBack -> TODO()
+        Action.ClickBack -> "Click back"
```
Comment (HlebMaliborski) [inline lib/src/main/java/fi/epicbot/toster/model/Action.kt:58]: Do we have a plan to emulate other clicks? Maybe pass parameter here as some click?

**Review by HlebMaliborski — APPROVED** [review]

File: lib/src/main/java/fi/epicbot/toster/model/SwipeMove.kt (hunk 1 above)
Comment (alexander-kulikovskii) [reply lib/src/main/java/fi/epicbot/toster/model/SwipeMove.kt:90]: sure :)

File: lib/src/main/java/fi/epicbot/toster/model/Action.kt (hunk 2 above)
Comment (alexander-kulikovskii) [reply lib/src/main/java/fi/epicbot/toster/model/Action.kt:58]: yeah, click on item with some id, if this id exists.
but not right now

**Pull request merged**